css/style.css

```diff
@@ -19,12 +20,38 @@ text-align: center;
   display: flex;
   justify-content: center;
   align-items: center;
-  letter-spacing: .5em;
-  font-family: 'Londrina Outline', cursive;
+  letter-spacing: .3em;
+  font-family: 'Skranji', cursive;
+  font-family: 'Londrina Solid', cursive;
   font-size: 6em;
   line-height: 2em;
   color: white;
-  text-emphasis: 900;
+  /* text-shadow: -8px 8px rgb(87, 130, 150); */
+}
+
+h1 > sup {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20pt;
+  margin-bottom: 25px;
+  text-shadow: none;
+}
+
+h3 {
+  font-size: 2em;
```

Comment:
Consider reducing h3 and h4 just a little, they still look a little big for the page.